cmd/kubeadm/app/constants/constants.go

```diff
@@ -31,6 +31,10 @@ const (
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
 
+	ServiceAccountKeyBaseName    = "sa"
+	ServiceAccountPublicKeyName  = "sa.pub"
```

Comment:
Yes, it does. I initially assumed that a cert is needed, but seems like just the public key is necessary, which is great